internal/pkg/agent/install/install.go

```diff
@@ -23,13 +24,16 @@ import (
 
 const (
 	darwin = "darwin"
+
+	elasticUsername  = "elastic-agent"
+	elasticGroupName = "elastic-agent"
 )
 
 // Install installs Elastic Agent persistently on the system including creating and starting its service.
-func Install(cfgFile, topPath string, pt *progressbar.ProgressBar, streams *cli.IOStreams) error {
+func Install(cfgFile, topPath string, nonRoot bool, pt *progressbar.ProgressBar, streams *cli.IOStreams) (string, string, error) {
```

Comment:
I know you mentioned it in the PR description that you're returning UID and GID as strings for future Windows support.  Consider mentioning that in this function's godoc as well for developers reading this code in the future without the context of this PR.

Reply:
I was able to change this behavior removing the need for these type of comments.